### PR TITLE
feature/WOD-15_Order_100test_coverage

### PR DESCRIPTION
**Pull request by LokmaneKrizou**
Link to the Code coverage: https://386-295777728-gh.circle-artifacts.com/0/reports-sdk/com.realifetech.sdk.sell.order.data.datasource/index.html

### CODE REVIEW

**Comment by LokmaneKrizou** [comment]
Screenshot 2021-09-07 at 09 45 09

**Comment by LokmaneKrizou** [comment]
Link to the Code coverage: https://386-295777728-gh.circle-artifacts.com/0/reports-sdk/com.realifetech.sdk.sell.order.data.datasource/index.html

File: sdk/src/main/java/com/realifetech/sdk/sell/order/data/datasource/OrderBackendDataSource.kt [hunk 1]
```diff
@@ -57,8 +61,10 @@ class OrderBackendDataSource @Inject constructor(private val apolloClient: Apoll
                 .build()
             response.enqueue(object : ApolloCall.Callback<GetMyOrderByIdQuery.Data>() {
                 override fun onResponse(response: Response<GetMyOrderByIdQuery.Data>) {
-                    val order = response.data?.getMyOrder?.fragments?.fragmentOrder?.asModel
-                    callback.invoke(null, order)
+                    response.data?.getMyOrder?.fragments?.fragmentOrder?.let {
+                        return callback.invoke(null, it.asModel)
```
Comment (sjulia7) [inline sdk/src/main/java/com/realifetech/sdk/sell/order/data/datasource/OrderBackendDataSource.kt:65]: why do you need to return this callback here, it wasn't returned before?

**Review by sjulia7 — APPROVED** [review]

File: sdk/src/main/java/com/realifetech/sdk/sell/order/data/datasource/OrderBackendDataSource.kt (hunk 1 above)
Comment (LokmaneKrizou) [reply sdk/src/main/java/com/realifetech/sdk/sell/order/data/datasource/OrderBackendDataSource.kt:65]: no i just re formatted this part so i can cover all the scenarios requested by jacoco test report

Comment (sjulia7) [reply sdk/src/main/java/com/realifetech/sdk/sell/order/data/datasource/OrderBackendDataSource.kt:65]: but now there's a return `callback.invoke(null, it.asModel)` , and before it was just  `callback.invoke(null, order)`, this is what I don't get

**Review by Fabio-Insolia-LiveStyled — APPROVED** [review]

**Pull request merged**